JavaScriptSPA/authRedirect.js

```diff
@@ -12,22 +12,22 @@ function authRedirectCallBack(error, response) {
     console.log(error);
   } else {
     if (response.tokenType === "id_token") {
-      console.log('id_token acquired at: ' + new Date().toString());
+      console.log("id_token acquired at: " + new Date().toString());
       myMSALObj.getAccount();
       getTokenRedirect(tokenRequest);
     } else if (response.tokenType === "access_token") {
-        console.log('access_token acquired at: ' + new Date().toString());
+        console.log("access_token acquired at: " + new Date().toString());
         accessToken = response.accessToken;
-        logMessage("Request made to Web API:")
-        if (accessToken === null || accessToken === undefined) {
+        logMessage("Request made to Web API:");
+        if (accessToken) {
```

Comment:
Before this was checking if `accessToken` was falsey, now it appears to be checking if it is truthy, that intended?

Reply:
that's right, the intention was to check for accessToken before calling the api, or acquire it if it is not (down below). The 'if' statement for the two was mixed up..